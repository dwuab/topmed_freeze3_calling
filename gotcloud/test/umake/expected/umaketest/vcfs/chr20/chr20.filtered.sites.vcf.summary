--------------------------------------------------------------------------------------------------------------
              FILTER	#SNPs	#dbSNP	%dbSNP	%CpG	%CpG	%Known	%Novel	%nCpG-K	%nCpG-N	%HM3	%HM3
                    				Known	Novel	Ts/Tv	Ts/Tv	Ts/Tv	Ts/Tv	sens	/SNP
--------------------------------------------------------------------------------------------------------------
              INDEL5	8	7	87.5	0.0	100.0	NA	0.00	NA	NA	0.003	12.500
          INDEL5;SVM	3	3	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
                PASS	1003	976	97.3	15.8	3.7	2.46	2.38	2.24	2.25	0.571	21.137
                 SVM	103	45	43.7	17.8	1.7	2.21	0.61	1.85	0.63	0.000	0.000
--------------------------------------------------------------------------------------------------------------
              FILTER	#SNPs	#dbSNP	%dbSNP	%CpG	%CpG	%Known	%Novel	%nCpG-K	%nCpG-N	%HM3	%HM3
                    				Known	Novel	Ts/Tv	Ts/Tv	Ts/Tv	Ts/Tv	sens	/SNP
--------------------------------------------------------------------------------------------------------------
              INDEL5	11	10	90.9	0.0	100.0	NA	0.00	NA	NA	0.003	9.091
                PASS	1003	976	97.3	15.8	3.7	2.46	2.38	2.24	2.25	0.571	21.137
                 SVM	106	48	45.3	16.7	1.7	2.43	0.61	2.08	0.63	0.000	0.000
--------------------------------------------------------------------------------------------------------------
                PASS	1003	976	97.3	15.8	3.7	2.46	2.38	2.24	2.25	0.571	21.137
                FAIL	114	55	48.2	14.5	3.4	2.93	0.59	2.62	0.63	0.003	0.877
               TOTAL	1117	1031	92.3	15.7	3.5	2.48	0.91	2.25	0.93	0.573	19.069
------------------------------------------------------------------------------------------------------------
 MultiAllele Ref/Alt	0
  Repeated Positions	0
       TOTAL SKIPPED	0
------------------------------------------------------------------------------------------------------------
