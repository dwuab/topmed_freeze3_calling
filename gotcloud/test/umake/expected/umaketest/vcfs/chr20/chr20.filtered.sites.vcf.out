<gotcloud_root>/bin/invNorm --in <outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.raw --out <outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.norm

Available Options
                Input options : --in [<outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.raw],
                                --ignore
               Output Options : --out [<outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.norm],
                                --digits [3], --rank, --verbose
   Output compression Options : --plain [ON], --bgzf, --gzip

invNorm 1.1.1 -- Column-wise inverse-normal transformation of matrix
(c) 2011 Hyun Min Kang and Goo Jun

Reading positive examples from <gotcloud_root>/test/chr20Ref/1000G_omni2.5.b37.sites.chr20.PASS.vcf.gz 
Reading positive examples from <gotcloud_root>/test/chr20Ref/hapmap_3.3.b37.sites.chr20.vcf.gz 
Positive samples: 287, Negative samples: 37
DP	MQ	NS	AN	AB	AZ	FIC	SLRT	LBS	OBS	STR	STZ	CBR	CBZ	IOR	IOZ	AOI	AOZ	LQR	MQ0	MQ10	MQ20	MQ30	QUAL
<gotcloud_root>/bin/svm-train -s 0 -t 2 <outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.labeled.svm <outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.svm.model
*
optimization finished, #iter = 95
nu = 0.071772
obj = -11.741298, rho = 0.288957
nSV = 45, nBSV = 3
Total nSV = 45
<gotcloud_root>/bin/svm-predict <outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.svm <outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.svm.model <outdir_path>/vcfs/chr20/chr20.filtered.sites.vcf.svm.pred
Accuracy = 0% (0/1117) (classification)
